pkg/overlay/cache.go

```diff
@@ -78,42 +119,63 @@ func (cache *Cache) Inspect(ctx context.Context) (storage.Keys, error) {
 }
 
 // List returns a list of nodes from the cache DB
-func (cache *Cache) List(ctx context.Context, cursor storj.NodeID, limit int) ([]*pb.Node, error) {
+func (cache *Cache) List(ctx context.Context, cursor storj.NodeID, limit int) (_ []*pb.Node, err error) {
```

Comment:
why the underscored return values?

Reply:
because there's no need to give them names :)

Reply:
ok lol